contracts/tribute/src/contract.rs

```diff
@@ -335,6 +326,62 @@ fn refund_tribute(
         }))
 }
 
+struct TopNProposalInfo {
+    pub top_n_proposal: Option<Proposal>,
+    pub is_above_voting_threshold: bool,
+}
+
+impl TopNProposalInfo {
+    fn are_tributes_claimable(&self) -> Result<Proposal, ContractError> {
+        match self.top_n_proposal.as_ref() {
+            None => Err(ContractError::Std(StdError::generic_err(
+                "User voted for proposal outside of top N proposals",
+            ))),
+            Some(proposal) => {
+                if !self.is_above_voting_threshold {
+                    return Err(ContractError::Std(StdError::generic_err(
+                        "Tribute can not be claimed because the proposal received less voting percentage than required to deploy the liquidity.")));
```

Comment:
I would abstract away a bit the reasoning behind it and just say "Tribute not claimable: Proposal received less voting percentage than threshold: X required, but is Y"

for our code, the fact that this is about deploying liquidity shouldn't matter, our contract just has proposals that people vote on and will get tribute, or won't get tribute